config_defaults_inc.php

```diff
@@ -1413,6 +1413,12 @@
  */
 $g_default_bug_view_status = VS_PUBLIC;
 
+/**
+ * Default Bug dependent
+ * @global integer $g_default_bug_dependant
+ */
+$g_default_bug_dependant = BUG_DEPENDANT;
```

Comment:
@waltergar There should be no need for us to add config options since we are trying to have parity with the existing system that didn't need such config.